src/messages/layouts/MessageLayout.cpp

```diff
@@ -112,13 +112,13 @@ bool MessageLayout::layout(int width, float scale, MessageElementFlags flags)
     return true;
 }
 
-void MessageLayout::actuallyLayout(int width, MessageElementFlags flags)
+void MessageLayout::actuallyLayout(int width, MessageElementFlags mflags)
```

Comment:
Same as above but here it's a different name for no reason?!

Reply:
Sorry for the inconsistency, I'm happy to rename it.